internal/cmd/builds/get.go

```diff
@@ -0,0 +1,81 @@
+package builds
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/saucelabs/saucectl/internal/build"
+	cmds "github.com/saucelabs/saucectl/internal/cmd"
+	"github.com/saucelabs/saucectl/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+func GetCommand() *cobra.Command {
+	var out string
+	var byJob bool
+	var buildSource string
+
+	cmd := &cobra.Command{
+		Use:          "get",
```

Comment:
Required positional arg should be included in the usage:

```suggestion
		Use:          "get <buildID>",
```

Reply:
I think this is a good idea. I did go ahead and do the same for `builds list` though. Thoughts?